app/ante/eth.go

```diff
@@ -182,6 +183,7 @@ func (egcd EthGasConsumeDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simula
 		if err != nil {
 			return ctx, sdkerrors.Wrap(err, "failed to unpack tx data")
 		}
+		gasWanted += txData.GetGas()
```

Comment:
`GetGas()` returns the amount of gas `wanted` (the equivalent of `gasLimit` in Ethereum) - https://github.com/cosmos/cosmos-sdk/blob/5d86db3ce593bcc8c8694f7f83418925f7e9568a/types/tx/types.go#L128-L129

As per the spec linked in the description: `the anteHandler returns a maximum limit of gas the transaction can consume during execution called GasWanted. The actual amount consumed in the end is denominated GasUsed`